assets/css/campaign.css

```diff
@@ -0,0 +1,34 @@
+/* Modify Iris ColorPicker */
+.palette-only .iris-picker {
+    height: auto !important;
+    width: auto !important;
+    padding: 0 !important;
+    margin-right: 25px;
+}
+
+.palette-only .iris-picker .iris-picker-inner {
+    display: none !important;
+}
+.palette-only .wp-color-picker {
+    pointer-events: none;
+}
+.palette-only .wp-picker-input-wrap {
+    display: none !important;
+}
+.palette-only .wp-picker-holder {
+    max-width: 400px;
+}
+.palette-only .iris-picker .iris-palette {
+    margin: 0;
+    cursor: pointer;
+    display: inline-block !important;
+    height: 30px !important;
+    width: 30px !important;
+    float: none;
+  }
+.palette-only .iris-palette-container {
+    position: relative !important;
+    left: 0 !important;
+    bottom: 0 !important;
+    padding: 8px 10px 4px;
+  }
```

Comment:
Indentation seems of in the last two rules. Is this code here in order to adjust some part of the administration dashboard?

Reply:
Fixed. Yes it's for the color picker in the dashboard. I was trying to disable a part of the color picker used in CMB2, but it inherits from Wordpress's default colorpicker and doesn't come with that setting by default, so I figured it would be less messy to just hide it and fix the appearance a bit.